### PR TITLE
[PIM-6374] Add column module

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: features/product/edit_product_and_action.feature [hunk 1]
```diff
@@ -13,18 +13,6 @@ Feature: Product edition clicking on another action
     And I am on the products page
     And I display the columns SKU, Name, Description and Family
 
-  Scenario: Successfully edit a product and back to the grid
```
Comment (juliensnz) [inline features/product/edit_product_and_action.feature:16]: why do we remove this test?

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml [hunk 2]
```diff
@@ -14,6 +14,11 @@ extensions:
         aclResourceId: pim_enrich_product_edit_attributes
         position: 100
 
+    pim-product-edit-form-column:
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml]: sidebar seems more appropriated?

Comment (juliensnz) [reply src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml]: Or at least left column?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            events: {
+                'click .AknColumn-collapseButton': '_toggleColumn'
+            },
+
+            /**
+             * {@inheritdoc}
+             */
+            render: function () {
+                this.$el.html(this.template({
+                    labelInfos: __('pim_enrich.entity.product.infos')
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js]: should be injected by conf :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                }));
+
+                this.renderExtensions();
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js]: you can also call parent render

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/secondary-actions.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+             * Each extension represents a secondary action available for the user.
+             *
+             * {@inheritdoc}
+             */
+            render: function () {
+                this.$el.empty();
+
+                if (!_.isEmpty(this.extensions)) {
+                    this.$el.html(this.template({
+                        titleLabel: 'Other actions'
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/secondary-actions.js]: please use a translation key

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/column.html [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<div class="AknColumn-inner">
+    <div class="AknColumn-innerTop" data-drop-zone="navigation"></div>
+</div>
+<div class="AknColumn-collapseButton"></div>
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/column.html:4]: could be a span no?

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/secondary-actions.html [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<div class="AknSecondaryActions-button" data-toggle="dropdown"></div>
+<div class="AknDropdown-menu AknDropdown-menu--right" data-drop-zone="secondary-actions">
+    <div class="AknDropdown-menuTitle"><%= titleLabel %></div>
```
Comment (juliensnz) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/secondary-actions.html]: always use <%- please

**Review by juliensnz — COMMENTED** [review]
Almost good to merge :)

File: features/product/edit_product_and_action.feature (hunk 1 above)
Comment (pierallard) [reply features/product/edit_product_and_action.feature:16]: There is no more "save and back to grid" button (and this test was testing it)

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml (hunk 2 above)
Comment (pierallard) [reply src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml]: "Sidebar" is the name of the library used to display the tree, I would use another name. Left column is ok.

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/column.html (hunk 6 above)
Comment (pierallard) [reply src/Pim/Bundle/EnrichBundle/Resources/public/templates/form/column.html:4]: Changes nothing due to CSS. IMHO, more a DIV than a SPAN.

**Comment by tamarasaurus** [comment]
GTM

**Review by tamarasaurus — APPROVED** [review]

**Pull request merged**